masterlist.yaml

```
@@ -15906,16 +15924,18 @@ plugins:
     clean:
       - crc: 0xB4C1E773
         util: 'SSEEdit v4.0.3'
-  - name: 'ClockworkTitleRemover.esp'
-    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/47087/' ]
+  - name: 'ClockworkSuperiorSorting.esp'
+    req: [ *PapyrusExtender, *SKSE ]
+    msg:
+      - <<: *scriptExtenderRequiredScripts
     clean:
-      - crc: 0x913191E9
+      - crc: 0x80123223
         util: 'SSEEdit v4.0.3'
```

I get a different checksum, is this a future version of the plugin?

```suggestion
      - crc: 0x9E78E8D9
        util: 'SSEEdit v4.0.4'
```